tutorials/scripting/singletons_autoload.rst

```diff
@@ -94,6 +94,27 @@ be accessed directly in GDScript, without requiring ``get_node()``:
 
    PlayerVariables.health -= 10
 
+The **Enable** column has no effect in C# code. However, if the singleton is a
+C# script, a similar effect can be achieved by adding a static property:
+
+.. tabs::
+ .. code-tab:: csharp
+
+    public static PlayerVariables Instance { get; private set; }
+
+    public override void _Ready()
+    {
+        Instance = this;
+    }
```

Comment:
Without the class definition I'm worried that users may not understand where this code snippet is supposed to go.

Reply:
Hmm, agreed. I'll fill it out. Maybe it could be declared beforehand (for GDScript too) but writing out the whole C# class here at least makes it totally clear.